doc/api/vm.md

```diff
@@ -406,10 +406,12 @@ console.log(script.sourceMapURL);
 added:
  - v13.0.0
  - v12.16.0
+changes:
+ - version: REPLACEME
+   pr-url: https://github.com/nodejs/node/pull/57513
+   description: Marking the API stable.
 -->
 
-> Stability: 1 - Experimental
```

Comment:
There are still todos in https://github.com/nodejs/node/issues/37648

Reply:
In addition to that, source phase imports (e.g. https://github.com/nodejs/node/pull/56919) essentially add a new requirement to the vm module, and it is still waiting for patches included in [V8 13.4](https://github.com/nodejs/node/pull/57114). I strongly believe vm modules should not be marked as stable right now.

Reply:
It might make sense to remove vm modules from this PR so the rest can land sooner than later?  Other than this one I think the rest seem non-controvertial.